content/en/Getting started/glossary.md

```diff
@@ -109,7 +111,7 @@ with white box and black box testing.
 
 ## Known Vulnerability
```

Comment:
I have been tending to use "documented" or "published" instead of "known", although that is I think more of the industry standard phrasing.

It just always makes me think "known by whom?", especially when it's not something I know offhand.

Reply:
See my update to new line 116